README.md

```diff
@@ -609,6 +626,56 @@ have been saved by the Primary.
 ```
 
 
+# Partial-Verification Secondaries
+The Secondaries described and employed above ran full verification, employing
+the full suite of TUF and Uptane security checks. Uptane provides for a
+less demanding alternative: partial-verification Secondaries. These perform
+a few checks instead of the full set, at a measured cost to security, allowing
+weaker ECUs to still operate as Uptane-conformant Secondaries.
+
+The distinction is defined
+[in Section 8.3 of the Implementation Specification](https://docs.google.com/document/d/1wjg3hl0iDLNh7jIRaHl3IXhwm0ssOtDje5NemyTBcaw/edit?pli=1#heading=h.22u629s8u37q).
+Briefly, partial verification entails one signature check to validate metadata,
+and one signature check whenever the minimum time is ratcheted forward (for
+metadata expiration purposes). Partial-verification Secondaries need to know
+the Director's Targets role public key and the Timeserver's public key.
```

Comment:
Is there a mechanism for acquiring the public key, or is it left to the implementer?  Does the reference implementation provide a way for a PVS to acquire the Director's targets public key?

These are only questions that I ask myself while reading these standalone README edits.